microceph/ceph/services.go

```diff
@@ -83,6 +100,26 @@ func RestartCephService(service string) error {
 	return nil
 }
 
+func getUpRgws() (common.Set, error) {
+	// check if rgw was up for atleast 2 seconds.
+	rgwSocketFiles := common.FilterFilesInDir(constants.RgwSockPattern, constants.GetPathConst().RunPath)
+	for _, file := range rgwSocketFiles {
+		age := common.GetFileAge(file)
+		if age < constants.RgwRestartAgeThreshold {
+			logger.Info(fmt.Sprintf("File %s age is %f (< 2)", file, age))
```

Comment:
Nit: the log message says `(< 2)` but should maybe refer to the constants.RgwRestartAgeThreshold